bundle/manifests/apps.nvidia.com_nemodatastores.yaml

```diff
@@ -65,40 +65,60 @@ spec:
                 type: array
               dataStoreParams:
                 properties:
-                  appVersion:
+                  datastoreConfigSecret:
                     type: string
-                  databaseHost:
+                  datastoreInitSecret:
                     type: string
-                  databasePort:
+                  datastoreInlineConfigSecret:
                     type: string
-                  databaseURL:
+                  datastoreSettingsSecret:
                     type: string
                   dbSecret:
                     type: string
-                  envConfigmap:
+                  giteaAdminSecret:
                     type: string
-                  envSecret:
+                  lfsJwtSecret:
                     type: string
-                  giteaEndpoint:
-                    type: string
-                  giteaSecret:
-                    type: string
-                  initContainerCommand:
-                    items:
-                      type: string
-                    type: array
-                  initContainerImage:
+                  objStoreSecret:
                     type: string
+                  persistence:
+                    properties:
+                      accessModes:
+                        items:
+                          type: string
+                        type: array
+                      claimName:
+                        type: string
+                      create:
+                        type: boolean
+                      enabled:
+                        type: boolean
+                      labels:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      size:
+                        type: string
+                      storageClass:
+                        type: string
+                    required:
+                    - claimName
+                    - create
+                    - enabled
+                    - size
```

Comment:
size is not a required param if this is an existing PVC. Same with claimName, if creation is enabled, we can add with default claimname based on the CR instance name.